components/playlist/browser/media_detector_component_installer.cc

```diff
@@ -136,6 +137,10 @@ MediaDetectorComponentInstallerPolicy::GetInstallerAttributes() const {
   return update_client::InstallerAttributes();
 }
 
+bool MediaDetectorComponentInstallerPolicy::IsBraveComponent() const {
```

Comment:
@goodov Could we check `IsBraveComponent() == true` in `BraveComponent::Register()` pipeline?

Reply:
I'd like to avoid touching anything related to `BraveComponent` here.